src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public int getQuantity(int defaultQuantity, int fruitQuantity) {
+        return defaultQuantity + fruitQuantity;
+    }
```

Comment:
The class `BalanceOperation` should be placed in the `strategy` package according to the checklist. Please ensure that your project structure adheres to the specified package organization.